src/contention_detectors/signal_analyzers/drop.hpp

```diff
@@ -131,6 +97,49 @@ class SignalDropAnalyzer : public SignalAnalyzer {
    */
   void shiftBasePoints();
 
+  //! int64_t
+  //! How far in the past we look.
+  void setCfgWindowSize(int64_t cfgWindowSize) {
+    SignalDropAnalyzer::cfgWindowSize = cfgWindowSize;
```

Comment:
Is SignalDropAnalyzer::cfgWindowSize is static? Looks like ;)
You should rather do windowSize = _windowsSize;